.travis.yml

```diff
@@ -1,36 +1,30 @@
 language: python
+python:
+  - 2.7
+  - 3.5
+  - 3.6
+  - 3.7
+  - 3.8
+  - "nightly"
 cache: pip
-dist: xenial
+dist: bionic
+install:
+  - pip install tox-travis codecov
+script:
+  - tox
+after_success:
+    - codecov
 
-matrix:
+jobs:
+  fast_finish: true
   include:
-    - name: Check black formatting
-      python: 3.7
-      env: TOXENV=black
-    - name: Check flake8 on Python 2
-      python: 2.7
-      env: TOXENV=flake8
-    - name: Check flake8 on Python 3
-      python: 3.7
-      env: TOXENV=flake8
-    - name: Check license headers
-      python: 3.7
-      env: TOXENV=license
-    - python: 2.7
-      env: TOXENV=py27
-    - python: 3.4
-      env: TOXENV=py34
-    - python: 3.5
-      env: TOXENV=py35
-    - python: 3.6
-      env: TOXENV=py36
     - python: 3.7
-      env: TOXENV=py37
-    - python: 3.8
-      env: TOXENV=py38
 
-install:
-  - pip install tox
+    - python: 3.7
```

Comment:
I liked having the `name` fields on the linting / license checks as these make the travis test results easier to consume. Does `jobs` support that too?

Reply:
See what you mean completely! Checking it.

Reply:
Yeah, previously it was jobs as well, I dropped the names by an oversight, now added all of them back.